src/assets/styles/sass/styles.scss

```diff
@@ -0,0 +1,66 @@
+.projects-listing {
+display: grid;
+grid-template: repeat(2, 1fr) / repeat(3, 1fr);
+place-items: center;
+column-gap: 1rem;
+row-gap: 1rem;
+}
+
+.project-card {
+    height: 100%;
+    width: 100%;
+    border: 1px solid rgb(213, 213, 213);
+    position: relative;
+
+    h3 {
+        text-align: center;
+    }
+}
+
+.project-bg {
+    background: linear-gradient(to right, rgba(255, 0, 0, 0.7), rgba(0, 128, 0, 0.5), rgba(0, 0, 255, 0.7), rgba(128, 0, 128, 0.7), rgba(0, 0, 0, 0.7), rgba(245, 241, 4, 0.7)), url("../../img/project-screenshots/city-info-screenshot.png") center center no-repeat;
+    background-size: contain;
+    height: 90%;
+    width: 100%;
+    display: grid;
+    grid-template: 5fr 0.5fr / 1fr;
+    place-items: center;
+}
+
+.project-card-links {
+    display: grid;
+    grid-template: 1fr / repeat(2, 1fr);
+    place-items: center;
+    column-gap: 1px;
+    height: 100%;
+    width: 100%;
+    background-color: rgba(255, 255, 255, 0.9);
+    border-radius: 4px;
+
+    a {
+        text-align: center;
+        text-decoration: none;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/manual/descholar-ceo/portfolio/pullRequest?prid=5731031)